action.yaml

```diff
@@ -0,0 +1,59 @@
+# action.yml
+name: "codacy-analysis-cli"
+description: "Execute Codacy code analysis using your remote Codacy configuration"
+branding:
+  icon: 'check'
+  color: 'gray-dark'
+inputs:
+  verbose:
+    required: false
+  project-token:
+    required: true
```

Comment:
I would put this as not required since the CLI can run without it. Even tough it will not provide results based on the remote configs it will show the defaults for the tools that allow it.